atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2016,7 +2012,6 @@ void WebContents::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("beginFrameSubscription", &WebContents::BeginFrameSubscription)
       .SetMethod("endFrameSubscription", &WebContents::EndFrameSubscription)
       .SetMethod("startDrag", &WebContents::StartDrag)
-      .SetMethod("setSize", &WebContents::SetSize)
```

Comment:
Please add this to `docs/api/breaking-changes.md` in `3-0-x` so that we follow the policy of warning one major release before removal